airflow/models.py

```diff
@@ -3434,8 +3434,14 @@ def sub_dag(self, task_regex, include_downstream=False,
         based on a regex that should match one or many tasks, and includes
         upstream and downstream neighbours based on the flag passed.
         """
-
-        dag = copy.deepcopy(self)
+        try:
+            dag = copy.deepcopy(self)
```

Comment:
I thought the issue is due to attribute fail deep copy? why we change on the subdag method?